src/Units/UnitContext.java

```diff
@@ -54,6 +68,8 @@ public UnitContext(Unit u)
     mods.addAll(u.getModifiers());
     initModel();
   }
+  // Note: the Commander argument is here as a placeholder for Army
+  // The plan is for Unit to have an Army reference, and UnitModel to have the Commander reference
```

Comment:
Why not give Commander the Army reference?

Reply:
`Commander` will also have the `Army`, but I think `Commander` may not have a list of units anymore (since that becomes a nigh-duplicate source of truth).
That means that `Army` owns units, `Commander` owns `UnitModel`s, and `Unit`s get references to them.

It just seems like the "right way" to structure the data, to me, but I'm certainly willing to hear counterarguments.

Reply:
I'd probably leave the unit list with the commander. An Army can iterate through all units via the commanders, and each commander still needs to iterate through their own units (since those are the ones they can move). Unless I'm missing something in your design. Army owns the COs, who own the units.

Reply:
> An Army can iterate through all units via the commanders

Fair point, though a nested for loop (for each commander, for each unit) seems a bit more complex than a for with a nested if (for each unit, if CO == x)
> each commander still needs to iterate through their own units (since those are the ones they can move)

`Army` determines what is movable during a given turn. For example, Wally wouldn't necessarily care which CO a given unit belonged to; he just wants to move every unit in his `Army`.
We could still end up having COs iterating through their own units for `turnInit`, D2D, or power reasons... but I was thinking most everything would be handled generically through `Army`.
> Unless I'm missing something in your design. Army owns the COs, who own the units.

Either object graph is defensible; they contain basically the same amount of info and duplication thereof:
![image](https://user-images.githubusercontent.com/8674241/143673910-210e2e6f-46d7-4be4-bdd6-addd639c5caf.png)
(the lines are supposed to be grabby stick-figure hands)